.github/workflows/publish_gui_nightly.yml

```
@@ -0,0 +1,95 @@
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Build
+      run: |
+        dotnet build UndertaleModTool --no-restore
+        dotnet build UndertaleModToolUpdater --no-restore
+    - name: Publish ${{ matrix.os }} GUI
+      run: |
+        dotnet publish UndertaleModTool -c ${{ matrix.configuration }} -r win-x64 --self-contained ${{ matrix.bundled }} -p:PublishSingleFile=${{ matrix.singlefile }} --output ${{ matrix.os }}
+        dotnet publish UndertaleModToolUpdater -c ${{ matrix.configuration }} -r win-x64 --self-contained ${{ matrix.bundled }} -p:PublishSingleFile=false --output ${{ matrix.os }}/Updater
+    - name: Copy external files
+      run: |
+        cp ./README.md ./${{ matrix.os }}
+        cp ./SCRIPTS.md ./${{ matrix.os }}
+        cp ./LICENSE.txt ./${{ matrix.os }}
+    - name: Create zip for nightly release Windows GUI
+      run: |
+        mkdir artifacts
+        7z a -tzip artifacts/GUI-${{ matrix.os }}-isBundled-${{ matrix.bundled }}-isSingleFile-${{ matrix.singlefile }}.zip ${{ matrix.os }} -mx0 
+
+  build_cli:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        configuration: [Debug]
+        bundled: [true, false]
+        include:
+          - os: ubuntu-latest
+            rid: linux-x64
+          - os: macOS-latest
+            rid: osx-x64
+          - os: windows-latest
+            rid: win-x64
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.x
+    - name: Restore dependencies
+      run: dotnet restore UndertaleModCli
+    - name: Build
+      run: dotnet build UndertaleModCli --no-restore
+    - name: Publish ${{ matrix.os }} CLI
+      run: dotnet publish UndertaleModCli -c ${{ matrix.configuration }} -r ${{ matrix.rid }} --self-contained ${{ matrix.bundled }} -p:PublishSingleFile=false --output CLI-${{ matrix.os }}
+    - name: Copy external files
+      run: |
+        cp ./README.md ./CLI-${{ matrix.os }}/
+        cp ./LICENSE.txt ./CLI-${{ matrix.os }}/
+    - name: Create zip for nightly release CLI
+      run: |
+        mkdir artifacts
+        7z a -tzip artifacts/GUI-${{ matrix.os }}-isBundled-${{ matrix.bundled }}-isSingleFile-${{ matrix.singlefile }}.zip ${{ matrix.os }} -mx0 
+
+  upload:
+
+    needs: [build_gui, build_cli]
+    runs-on: windows-latest
+    steps:
+    - name: Upload all artifact files
+      run: |
+        iwr -outf ghr_v0.14.0_windows_amd64.zip https://github.com/tcnksm/ghr/releases/download/v0.14.0/ghr_v0.14.0_windows_amd64.zip
+        7z x ghr_v0.14.0_windows_amd64.zip -y
+        "ghr_v0.14.0_windows_amd64/ghr" -recreate -n "Latest Build" -b "This is the latest build of UndertaleModTool. There may be bugs! If there are, please make an issue on GitHub or join the Underminers Discord for more help!" continuous artifacts/
```

Makes it a little bit clearer, that this is not intended for stable use.
```suggestion
        "ghr_v0.14.0_windows_amd64/ghr" -recreate -n "Latest Build" -b "This is an automatically updating **bleeding edge** build of UndertaleModTool. There *will* be bugs! If you encounter any, please make an issue on GitHub or join the Underminers Discord for more help!" continuous artifacts/
```